Localization/de-DE/Mods.PathOfTerraria.UI.hjson

```diff
@@ -297,4 +297,4 @@ Tips: {
 
 	Click to open skill tree
 	''' */
-// AreaLevel: "Area Level: "
+// AreaLevel: Area Level:
```

Comment:
Revert all `Mods.PathOfTerraria.UI.hjson` changes, we need to properly get rid of all localization lines with non-trimmed endings so we don't get annoyed with this.